x-pack/plugins/rule_registry/server/utils/create_persistence_rule_type_wrapper.ts

```diff
@@ -82,7 +82,13 @@ export const createPersistenceRuleTypeWrapper: CreatePersistenceRuleTypeWrapper
                 }
 
                 if (filteredAlerts.length === 0) {
-                  return { createdAlerts: [], errors: {} };
+                  return { createdAlerts: [], errors: {}, truncatedAlertsArray: false };
```

Comment:
Nit: `truncatedAlertsArray` sounds like it's an array, but it's a boolean. Mb `isAlertsArrayTruncated`?